lib/src/screens/contact/contact_list_page.dart

```diff
@@ -119,7 +119,9 @@ class ContactListPage extends BasePage {
 
         if (isCopied) {
           await Clipboard.setData(ClipboardData(text: contact.address));
-          await showBar<void>(context, S.of(context).copied_to_clipboard);
+          Navigator.of(context).pop(contact);
+          // await showBar<void>(context, S.of(context).copied_to_clipboard);
+         
```

Comment:
please revert those, and make the `contactListViewModel.isEditable` to false this will already handle the case without further modifications